internal/plugin/connectors/tcp/mssql/connector.go

```diff
@@ -33,69 +118,102 @@ func (connector *SingleUseConnector) Connect(
 
 	connector.clientConn = clientConn
 
-	err := connector.performPreLoginHandshake()
+	// Read the prelogin request from the client, but we don't need to do anything
+	// we just need to make sure the client isn't blocked
```

Comment:
I know this  was an existing comment, but it's a bit cryptic.  Let's change to:

```
Secretless _is_ the client with respect to the server, and there is nothing in the 
pre-login handshake that needs to be passed along.  Secretless simply reads
it from the client and throws it away, so that client can advance to the next 
stage of the process.  Otherwise the client would block forever waiting for its
pre-login handshake to be read.
```
cc/ @doodlesbykumbi Is there anything we should add to this for maximum
clarity?

Reply:
I agree that the new messaging is clearer. The original comment I wrote was inaccurate and the new one inherits from it. The truth is:

1. Secretless parses the The TDS7 prelogin-message from the client
2. Secretless incorporates content from (1) in the response to the client. At present, there's nothing from the prelogin-message that is needed for making requests to the server.
3. Secretless (doesn't currently but will do so very soon) parses the [TDS7 login](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/773a62b6-ee89-4c02-9e5e-344882630aac) from the client. This is where Secretless gets some client parameters that are then uses in generating Secretless' TDS7 login directed to the server.